pkg/internal/cluster/create/actions/config/config.go

```diff
@@ -143,6 +144,9 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 				if err := nodeutils.WriteFile(node, containerdConfigPath, patched); err != nil {
 					return errors.Wrap(err, "failed to write patched containerd config")
 				}
+				if err := waitForSystemdFullyUpAndRunning(node, time.Now().Add(5*time.Second)); err != nil {
+					return errors.Wrap(err, "failed to wait for the systemd to be fully up and running")
+				}
 				// restart containerd now that we've re-configured it
 				if err := node.Command("systemctl", "restart", "containerd").Run(); err != nil {
```

Comment:
what about making this:
```suggestion
				if err := node.Command("bash", "-c", `! systmctl is-system-running || systemctl restart containerd`).Run(); err != nil {
```

Reply:
(or possibly using a check to see if containerd is running, we only need to restart it if it's already running possibly with the unpatched config)